requirements-demo.txt

```diff
@@ -0,0 +1,6 @@
+bokeh==0.12.4
+ipywidgets
+matplotlib
+pandas
+seaborn
+widgetsnbextension
```

Comment:
If they break on newer versions, better to get issues about them.  As it stood before all these updates, the versions used were quite old from a pinned docker image. Great for a dev env, but it gives no indication of what the real install experience is going to be for users.